libs/blocks/merch-card/merch-card.js

```diff
@@ -173,23 +203,27 @@ const extractTags = (container) => [...container.querySelectorAll('p')]
 
 const addMerchCardGridIfMissing = (section, cardType) => {
   const el = section?.querySelector('.section-metadata');
+  const merchCardGrids = ['one-merch-card', 'two-merch-cards', 'three-merch-cards', 'four merch cards'];
   const directSection = section?.parentElement?.tagName === 'MAIN';
+  const merchCards = section.querySelectorAll('.merch-card');
   if (el) {
     const metadata = getMetadata(el);
     let styleClasses = [];
     styleClasses = metadata?.style?.text?.split(',').map((token) => token.split(' ').join('-')) ?? [];
     if (!styleClasses.some((styleClass) => /-merch-card/.test(styleClass))) {
       if (styleClasses.some((styleClass) => /-up/.test(styleClass) || directSection)) {
-        section.classList.add('three-merch-cards', cardType);
+        section.classList.add(`${merchCardGrids[merchCards.length - 1] || merchCardGrids[2]}`, cardType);
```

Comment:
this makes our code more fragile and IMO it might happen that it adds the wrong style.
I prefer deferring the task of adding the right style to GWP.

Reply:
As discussed, reverted this changes.